pkg/network/openshift_sdn.go

```diff
@@ -195,6 +195,12 @@ func nodeConfig(conf *netv1.NetworkConfig) (string, error) {
 			ClientCA:    "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
 			BindAddress: bindAddress + ":10251", // port is unused
 		},
+
+		// Openshift-sdn calls the CRI endpoint directly; point it to crio
+		KubeletArguments: legacyconfigv1.ExtendedArguments{
```

Comment:
So ultimately this is a rhcos / rhel / installer problem - why is crio not at the standard CRI path instead of having to know directly about crio?  @mrunalp for 4.0 it would be better if crio was at the path the kubelet defaults to.

Reply:
CRI is an interface, I should have to know it’s crio.

Reply:
Kubelet still special cases a crio sock for cadvisor , I’m not following Clayton’s comments.

Reply:
> Kubelet still special cases a crio sock for cadvisor , I’m not following Clayton’s comments.

CRI should allow the kubelet (and other consumers) to run on top of a given CRI implementation (e.g. CRI-O), without having to break the abstraction and care about which implementation it is.

Reply:
@wking i agree, that is not the state of the world at this time.  it will be 6mos or more before that is true.